### ISSUE TITLE
Move to updates

### ISSUE THREAD

**Issue by SoraKohaku** [cryptonotefoundation/cryptonote#268]
i just wanna said people keep post  issue here. try this

https://github.com/FndNur1Labs/CryptonoteEvo

if you not like you can back here again.

Cryptonote Non-Official Project

Change to~

Note : my opinion[because we need upgraded source from oldest to new]

**Comment by TheDataPioneer** [comment]
It looks like your CryptonoteEVO project has been abandoned :'(  
It was a very good idea, at least.
But I suggest people just fork Monero, as it is a Cryptonote fork that has been kept  up to date.

**Comment by SoraKohaku** [comment]
Yep that why update. Im leave that and make gain using new engine called MuBdI
